src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -84,7 +84,7 @@ export class SplashScreen extends React.Component<SplashScreenProps> {
     const mainItems = this.createMainItems();
     const commonTaskItems = this.createCommonTaskItems();
     let recentItems = this.createRecentItems();
-    if (userContext.features.notebooksTemporarilyDown) {
+    if (useNotebook.getState().isPhoenix) {
```

Comment:
should this be "!useNotebook.getState().isPhoenix"

Reply:
No, according to previous flag when notebooks are down then we need to remove 'Notebooks' from recent activity. Now with this flag we need only when phoenix enabled. Removing this check so that default removes notebooks from recent activity